seed/java-sdk/http-head/src/main/java/com/seed/httpHead/resources/user/RawUserClient.java

```diff
@@ -49,9 +50,7 @@ public SeedHttpHeadHttpResponse<Void> head(RequestOptions requestOptions) {
         }
         try (Response response = client.newCall(okhttpRequest).execute()) {
             ResponseBody responseBody = response.body();
-            if (response.isSuccessful()) {
-                return new SeedHttpHeadHttpResponse<>(null, response);
-            }
+            if (response.isSuccessful()) {}
```

Comment:
Same thing here - we should still be returning `SeedHeadHttpResponse<Void>` here.